ga4gh/frontend.py

```diff
@@ -228,6 +274,9 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(SECRET_KEY_LENGTH)
```

Comment:
We should add a comment here explaining what needs to be done here when we move to multiple processes. It's not clear to me how we're going to generate different keys in different processes and decide which one is the real key. Perhaps this should just be a config variable?